### ISSUE TITLE
Delete this code

### ISSUE THREAD

**Issue by itIsMaku** [matherion-leak2/the-bridge-v2#2]
Hi, @matherion-leak2,
I'd like you to stop committing under my email and take down that code immediately. This code does not belong to you and you do not have any permission to it. This code belongs to @VottusCode and me and you do not have the permission to upload it here. Take it off immediately or it will be solved differently.
Have a nice day,
itIsMaku

**Comment by VottusCode** [comment]
Indeed. Please see matherion-leak2/matherionity#1 for more detailed explanation.
